mysql/const.go

```diff
@@ -211,6 +211,8 @@ const (
 	SuperPriv
 	// CreateUserPriv is the privilege to create user.
 	CreateUserPriv
+	// CreateTablespacePriv is the privilege to create tablespace.
+	CreateTablespacePriv
```

Comment:
Please put this constant to the end to preserve the consistency across versions.

Reply:
OK